Atomic/backends/_docker.py

```diff
@@ -537,10 +549,34 @@ def add_string_or_list_to_list(list_item, value):
                                "{}s\n\n removes all containers based on an "
                                "image.".format(iobject.name, iobject.image_name, iobject.name, iobject.name,
                                                iobject.image_name, iobject.image_name, iobject.image_name))
+
+            requested_image = self.has_image(args.image)
             if iobject.running:
+                if args.replace:
+                    iobject = delete_existing_container(iobject, requested_image, args)
```

Comment:
Should we call this replace_existing_container?

delete_* returning something new seems weird.